kubernetes/schema_rbac.go

```diff
@@ -2,15 +2,17 @@ package kubernetes
 
 import (
 	"github.com/hashicorp/terraform/helper/schema"
+	"github.com/hashicorp/terraform/helper/validation"
 )
 
 func rbacRoleRefSchema(kind string) map[string]*schema.Schema {
 	return map[string]*schema.Schema{
 		"api_group": {
-			Type:        schema.TypeString,
-			Description: "The API group of the user. Always `rbac.authorization.k8s.io`",
-			Required:    true,
-			Default:     "rbac.authorization.k8s.io",
+			Type:         schema.TypeString,
+			Description:  "The API group of the user. Always `rbac.authorization.k8s.io`",
+			Required:     true,
+			Default:      "rbac.authorization.k8s.io",
```

Comment:
I think you should just make this optional so the `Default` can apply.

Reply:
I'd rather drop the default so people are aware of it. We're getting issues filed with examples that are wrong in that regard, so awareness of it is not what one would hope.